src/header.html

```diff
@@ -2,37 +2,42 @@
                 </svg>
-            </div>
-        </div>
+            </a>
 
-        <div class="wsusearch">
-            <nav>
-                <ul>
-                    <li><a href="http://academica.wayne.edu/" class="spf-nolink">Academica</a></li>
-                    <li><a href="http://wayne.edu/directory/" class="spf-nolink">Directory</a></li>
-                    <li><a href="http://wayne.edu/contact/" class="spf-nolink">Contact</a></li>
-                    <li class="aimhigher"><a href="http://wayne.edu/aimhigher/" class="spf-nolink">Aim Higher</a></li>
-                </ul>
-            </nav>
+            <div class="wsusearch">
+                <div class="wsulogin">
+                    <a href="https://login.wayne.edu">Login</a>
+                </div>
 
-            <form method="get" action="https://wayne.edu/search/">
-                <label for="q">Search:</label>
-                <input name="q" class="input-group-field" id="q" placeholder="Search..." type="text">
-                <button type="submit" class="button"><span class="search-icon"><span>Search</span></span></button>
-            </form>
+                <div class="wsusearchbar">
+                    <form method="get" action="/search/">
```

Comment:
For this version the action needs to be a fully qualified URL since this can be on any domain. `https://wayne.edu/search/`